libraries/ESP32/examples/RMT/Legacy_RMT_Driver_Compatible/Legacy_RMT_Driver_Compatible.ino

```diff
@@ -0,0 +1,52 @@
+/*
+ * This example demonstrates how to use the file build_opt.h to disable the new RMT Driver
+ * Note that this file shall be added the Arduino project 
+ * 
+ * If the content of this file changes, it is necessary to delete the compiled Arduino IDE cache
+ * It can be done by changing, for instance, the "Core Debug Level" option, forcing the system to rebuild the Arduino Core
+ * 
+ */
+
+#ifndef ESP32_ARDUINO_NEW_RMT_DRV_OFF
+
+// add the file "build_opt.h" to your Arduino project folder with "-DESP32_ARDUINO_NEW_RMT_DRV_OFF" to use the RMT Legacy driver
+#warning "ESP32_ARDUINO_NEW_RMT_DRV_OFF is not defined, using new RMT driver"
```

Comment:
Imho a warning for using a actual function is not correct.
It should be the other way around.

Reply:
This is part of the example, it is just a way to say "this example is intended to demonstrate how to use RMT Legacy Driver within Arduino Core 3.0.x + IDF 5.x"

Please elaborate futher.

Reply:
I would say: make it an error and no extra code. It should never fail, wright? except if we mess up platform.txt, in which case it will be used as notification that the feature broke.

Reply:
I have modified it. Please review it.